src/head/script.js

```diff
@@ -92,4 +96,109 @@ export default config => {
 		}());
 		/* eslint-enable */
 	}
+
+	// One Trust Cookie Management
+	if (config.oneTrust && config.oneTrust.enable) {
+		/* eslint-disable */
+		(function () {
+			// Main OneTrust script to display cookie notice and set user preferences
+			const p = document.getElementsByTagName('script')[0];
+			const s = document.createElement('script');
+			s.setAttribute('type', 'text/javascript');
+			s.setAttribute('data-domain-script', `${config.oneTrust.key}${config.oneTrust.domainSuffix}`)
+			s.src = `https://cdn.cookielaw.org/consent/${config.oneTrust.key}${config.oneTrust.domainSuffix}/otSDKStub.js`;
+			p.parentNode.insertBefore(s, p);
+		}());
+
+		// Follow up OneTrust function.
+		// This function is called when cookie preferences are changed on the OneTrust widget
+		window.OptanonWrapper = () => {
+			/**
+			 * Insert scripts if user has enabled OneTrust Category
+			 * These categories must match the categories for the specific cookies in the OneTrust admin interface
+			 * */
+
+			/** Category 'C0001'
+			* Strictly Necessary Cookies
+			* These cookies are necessary for the website to function and cannot be switched off in our systems.
+			* They are usually only set in response to actions made by you which amount to a request for services,
+			* such as setting your privacy preferences, logging in or filling in forms. You can set your browser
+			* to block or alert you about these cookies, but some parts of the site will not then work.
+			* These cookies do not store any personally identifiable information.
+			* */
+
+			/** Category 'C0002'
+			* Performance Cookies
+			* These cookies allow us to count visits and traffic sources so we can measure and improve the performance
+			* of our site. They help us to know which pages are the most and least popular and see how visitors move
+			* around the site. All information these cookies collect is aggregated and therefore anonymous. If you
+			* do not allow these cookies we will not know when you have visited our site, and will not be able to
+			* monitor its performance.
+			* */
+			if (config.enableAnalytics) {
+				if (config.enableGA && !optout) {
+					OneTrust.InsertHtml('', 'head', insertGoogleAnalytics, null, 'C0002');
+				}
+				if (config.enableSnowplow) {
+					OneTrust.InsertHtml('', 'head', insertSnowplow, null, 'C0002');
+				}
+				if (config.enableGTM && !optout) {
```

Comment:
GTM needs to be set as a Targeting tool. It's primary usage is to load up marketing tags.